src/core/lv2/Lv2SubPluginFeatures.cpp

```diff
@@ -144,8 +220,21 @@ QString Lv2SubPluginFeatures::displayName(
 QString Lv2SubPluginFeatures::description(
 	const Plugin::Descriptor::SubPluginFeatures::Key &k) const
 {
-	(void)k;
-	return QString::fromUtf8("description not implemented yet"); // TODO
+	auto mgr = Engine::getLv2Manager();
+	const LilvPlugin *plug = mgr->getPlugin(k.attributes["uri"]);
```

Comment:
```suggestion
	const LilvPlugin* plug = mgr->getPlugin(k.attributes["uri"]);
```

Minor style suggestion.

Reply:
We do not have a style convention for this stuff. I fear if I change this, either someone will tell me to change it consistent in the whole file, or will tell me that I should change it back :smiley:

Reply:
Well, i do get confused about this often, so that's why i change, and ask to change in review. I thought this was a part of the convention. Sorry for the confusion.